server/app/grading/grade.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+echo "Score 15"
```

Comment:
The output format for an autograder needs to match what comes out of OK, which includes scores for multiple partners. For example, running `ok -s` gives:

```
...
Score
    Partner 0: 18.0
```

and we should mandate this format for all autograder scripts. If a script ends in

```
Score
    A: 12.0
    B: 13.0
```

then two new records in the `Score` model should be created, one with `key=A,score=12` and one with `key=B,score13`.